sky/packages/sky/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:ui' as ui;
```

Comment:
can you elaborate on whether these classes' fields are supposed to be mutable or not, and why/why not? (maybe in a comment)